### PR TITLE
Update openebs/helm-operator for 1.5.0 release

### PR DESCRIPTION
**Pull request by ChandanSagar**
- Updated Makefile
 - Updated template cr yaml
 - Added 1.5.0 cluster service version yaml

Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>

### CODE REVIEW

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,497 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (ksatchit) [inline olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml]: I suppose file name should be changed?

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+
+    ## Getting Started
+    
+    * Try the quickstart [guide](https://github.com/openebs/helm-operator/blob/master/olm/README.md)
+
+    * To learn how to contribute, please read the [contribution guide](https://github.com/openebs/helm-operator/blob/master/CONTRIBUTING.md)
+
+    * OpenEBS welcomes your feedback and contributions in any form possible. [Join our Community](https://openebs.org/community)
+
+    ## Workarounds
```
Comment (ksatchit) [inline olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml]: I suppose this is not needed in 1.5?

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml~ [hunk 3]
```diff
@@ -0,0 +1,497 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (ksatchit) [inline olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml~]: why is this file needed at all ?

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (ChandanSagar) [reply olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml]: Changed the file name.

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml (hunk 2 above)
Comment (ChandanSagar) [reply olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml]: removed the work arounds.

File: olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml~ (hunk 3 above)
Comment (ChandanSagar) [reply olm/1.5.0/openebsoperator.v1.4.0.clusterserviceversion.yaml~]: removed the file.

File: olm/1.5.0/openebsoperator.v1.5.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
...RC62IYOo65bBWHj4OXn1HNy4UlI9FdOICA4XWgoRHPx8xUpm+szlyyE5b7jVTBWPQxmAUZVGMXVXoFsAfAhAmx0GixV0SyvMr0/ZYaharqhEnksh4EiBwwU6VBqJSMRzXdfR5nChJzhCFAXnZUIdgg6Ac6akEAEq9dqhw5DWLurTyzo6RK5Uaqh1foxbxvKODkQcR7RYC9SZogvA79phsFhBt8x1/gHAaXYYjlQ4YwyOtrY1EaKuqyOuUz4ida1SrIVJIZipI5OLiCA5Z64UHKgEwkU8X5+1YjkPlGLNkNjq/oADYJ7nqaWdHTYY7sT8EYC4HQaLFXTLXOVcWFf7lNHGIOK6MhWJOkfkjFeCzCKO4HVXTqOKrhMMAUTmnFUrqC0akcUgaFIOGYMyJgKAL+/oMO2xmCgrZR/y8ekD8E47DBYr6Ja5yj/bIajdkk5GfIUjy8jBlZI5XKCeM3TGGJQxCLVmAGF5d7de39cnssXmuNoZY1Bam1ypaACYld1d3BDZYLiT8zt2CCxW0C1zkesBXGWHoWZLGu2xmARjR0i34Byc8/oEHYAxxs0UCr6QUl24ZjXTxvBmnWdzxhAoZfZPTIbtqSQtSqdFYRr14RcQGwG81A6DxQq6ZS4RAfAROwy1o7RGKho1rutq/azbnThjRaqzKbgjJUayWWWUUpeeso6lopGGp6nhqM1HvlwWQRjKU/r6SHDO5nkZ10byFjsEFivolrnEnwFYYYehdqo54yIZiZA57BydMabrPewmIgRah+uXLtFrurtFptjUqHY4QuDA5GQ5FY2WVnV3VzYP9tFOlVcAWGKHwWIF3TIXWATg/9lhqI9KTXeHdcRjwGFWredIxuqUxUIQYFlHR+SCVav8QhDMSNOTQClnTW+PF3Ec2MpwNREF8Eo7DBYr6Ja5wP8DkLDDMA0YWEc8DgD6oJedM0b1X67yj7JSzBA13VouBAF6UimxqqvLaXKO+3zF1ne3WEG3zDq9AN5lh2F6hEqjPR7njucdqt0+XauaiEAzdI5tjEFbNAopBGwhmbq4DMAyOwwWK+iW2eSPrXU+fQKl0BaJsPZolEwld5t50omwFmk7yhgDEUEbY63z+pAAXmiHwWIF3TJbdGMB9jlvioVLBM91eHcqCQAExuAILlopsszGtE+bF9khsFhBt8wW7wSQtMPQIFE3xLqTSQ7ODYhg
```
Comment (ksatchit) [inline olm/1.5.0/openebsoperator.v1.5.0.clusterserviceversion.yaml]: Should be v0.0.5 ??

Comment (ChandanSagar) [reply olm/1.5.0/openebsoperator.v1.5.0.clusterserviceversion.yaml]: Updated the same.
